firefox-ios/firefox-ios-tests/Tests/ClientTests/TabTray/TabsPanelStateTests.swift

```diff
@@ -20,10 +20,20 @@ final class TabPanelStateTests: XCTestCase {
         super.tearDown()
         DependencyHelperMock().reset()
     }
-
+    // MARK: - TabPanelMiddlewareActionType
     func testTabsState_DidLoadTabPanel() {
+        testTabsState_newTabsIsEmptyHelper(actionType: TabPanelMiddlewareActionType.didLoadTabPanel)
```

Comment:
separating these methods out makes it a bit harder to read these tests. What do you think about keeping some code so that it follows our given, when, then format and we can create helper methods for any setup needed that might bloat the test method?